extensions/gitpod/package.json

```diff
@@ -0,0 +1,108 @@
+{
+	"name": "gitpod-desktop",
+	"displayName": "%displayName%",
+	"description": "%description%",
+	"publisher": "gitpod",
+	"version": "0.0.29",
+	"license": "MIT",
+	"preview": true,
+	"icon": "resources/gitpod.png",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/gitpod-io/vscode.git"
+	},
+	"bugs": {
+		"url": "https://github.com/gitpod-io/gitpod/issues"
+	},
+	"engines": {
+		"vscode": "^1.58.2"
+	},
+	"categories": [
+		"Other"
+	],
+	"keywords": [
+		"remote development",
+		"remote"
+	],
+	"extensionKind": [
+		"ui"
+	],
+	"activationEvents": [
+		"*",
+		"onCommand:gitpod.api.autoTunnel",
+		"onAuthenticationRequest:gitpod"
+	],
+	"contributes": {
+		"authentication": [
+			{
+				"label": "Gitpod",
+				"id": "gitpod"
+			}
+		],
+		"configuration": [
+			{
+				"title": "Gitpod",
+				"properties": {
+					"gitpod.serviceUrl": {
```

Comment:
why not just `host`? It is not only about URIs, but we scope all data of extension to some host and if this setting is chagned then we should unload data for old host and load for new host

Reply:
I agree `host` makes more sense here

Reply:
Changed it in https://github.com/gitpod-io/openvscode-server/pull/337/commits/606c2c5198a5321ad612581ef056b9dcd06e45f7, I think it could be written in a clearer way, but I think just inputting the hostname is better than a URL, since it is shorter and makes sense to have in some places (logging, keychain keys).

I am open to reverting it back to the URL inputting though, we could do some easy parsing.

Reply:
I'm fine with the rename (forgot it was suggested in the first place) but I'd much prefer to be a URL as before, that way I can just copy it form the browser and not think about it, now it will throw an error

Reply:
Thanks for the feedback, @jeanp413. I'll revert it back to a full URL.

Reply:
https://github.com/gitpod-io/openvscode-server/pull/337/commits/71b6ac84091c368dbd00adbe1f4311f0b8cb9ddf